operator/cmd/mesh/install.go

```diff
@@ -273,13 +273,14 @@ func DetectIstioVersionDiff(cmd *cobra.Command, tag string, ns string, kubeClien
 				icpTag = val
 			}
 		}
+		revision := manifest.GetValueForSetFlag(setFlags, "revision")
```

Comment:
We should get the `revision` from setFlags because the `revision` flag has been applied as alias before this: https://github.com/istio/istio/blob/27ec5c08e950633daa509eb82f9a2aa9a9c9cced/operator/cmd/mesh/install.go#L140

Reply:
`applyFlagAliases` function handles and assign `iArgs.revision`. so if the user provides `--set revision=1.8.0` or `--revision 1.8.0` it handles both the cases.